include/irr/builtin/glsl/sampling/cos_weighted.glsl

```diff
@@ -22,6 +22,11 @@ float irr_glsl_projected_hemisphere_remainder_and_pdf(out float pdf, in vec3 L)
 	return irr_glsl_projected_hemisphere_remainder_and_pdf(pdf,L.z);
 }
 
+float irr_glsl_projected_hemisphere_pdf(in float L_z)
```

Comment:
I hate duplicate code @Crisspl make sure that `float irr_glsl_projected_hemisphere_remainder_and_pdf(out float pdf, in float L_z)` uses `float irr_glsl_projected_hemisphere_pdf(in float L_z)`

Reply:
duplicate code

Reply:
still duplicate code